hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/BaseTableServicePlanActionExecutor.java

```diff
@@ -116,7 +116,7 @@ public Pair<Option<HoodieInstant>, Set<String>> getIncrementalPartitions(TableSe
     String rightBoundary = instantTime;
     // compute [leftBoundary, rightBoundary) as time window
     HoodieActiveTimeline activeTimeline = table.getActiveTimeline();
-    Set<String> partitionsInCommitMeta = table.getActiveTimeline().filterCompletedInstants().getCommitsTimeline().getInstantsAsStream()
+    Set<String> partitionsInCommitMeta = table.getActiveTimeline().findInstantsAfter(leftBoundary).filterCompletedInstants().getCommitsTimeline().getInstantsAsStream()
```

Comment:
why filtering the instant time by `leftBoundary`?

Reply:
@danny0405 Nice catch Danny.
At first I just wanted to filter out the left boundary instant itself, but obviously I called the API `findInstantsAfter `by mistake. Here we should get and filter the entire active time line complete instants